### ISSUE TITLE
on('stabilizing'), on('beginStabilizing'), on('endStabilizing')

### ISSUE THREAD

**Issue by klmdb** [almende/vis#322]
While we have an event to capture when the network stops stabilizing, we have no easy way to know when it starts stabilizing again or when it is busy stabilizing. 

These events would be usefull to me to bind behavior to nodes or edges moving around the canvas (overlaying a dom element bound to a node's location, for example).

Any chance to see these events added or should I try to hack a solution?

Regards,
klmdb

**Comment by AlexDM0** [comment]
Hi,

I've been looking into this. There has been a decently sized discussion here: https://github.com/almende/vis/issues/344

I tend to think this is actually not a bug, thought it should be documented decently.

TL;DR: the simulation starts on an update of the dataset/setOptions etc. If the simulation starts, stabilization starts (by definition). If we judge the network stabilized we stop the simulation and emit the stabilized event. This is done for every start/stop of the network. Longer story is in the issue I referenced.

I'd like to hear your opinion on what we should do. I like the "once" solution personally.

Regards,

Alex
